@navikt/aksel-icons/package.json

```
@@ -62,7 +62,7 @@
     "update-metadata": "node config/metadata.js",
     "write-packagejson": "echo '{\"type\": \"module\"}' > dist/react/esm/package.json",
     "build": "yarn copy && yarn create-icons && concurrently \"tsc\" \"tsc -p tsconfig.esm.json && tsc-alias -p tsconfig.esm.json && yarn write-packagejson\" && node config/cleanTypes.js && yarn update-metadata && yarn parse-svg",
-    "fetch-new:icons": "node config/figma/index.mjs",
+    "fetch-new:icons": "yarn tsx --env-file=.env config/figma/index.mjs",
```

Do we need `yarn` in the command?
```suggestion
    "fetch-new:icons": "tsx --env-file=.env config/figma/index.mjs",
```

---

Probably possible to change this on the other commands too, but not at all important ofc.